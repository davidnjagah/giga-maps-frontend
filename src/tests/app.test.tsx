import { describe, expect, test } from '@jest/globals';
import { render } from '@testing-library/react';

import App from '../app';

describe('App', () => {
  test('App snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
})
