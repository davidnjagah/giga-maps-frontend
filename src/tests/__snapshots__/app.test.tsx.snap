// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot 1`] = `
<DocumentFragment>
  <div
    class="cds--loading-overlay"
  >
    <div
      aria-atomic="true"
      aria-live="assertive"
      class="cds--loading"
    >
      <svg
        class="cds--loading__svg"
        viewBox="0 0 100 100"
      >
        <title>
          loading
        </title>
        <circle
          class="cds--loading__stroke"
          cx="50%"
          cy="50%"
          r="44"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
